tensorflow/core/kernels/image/non_max_suppression_op.cc

```diff
@@ -192,14 +192,17 @@ void DoNonMaxSuppressionOp(OpKernelContext* context, const Tensor& scores,
   }
 
   T scale = static_cast<T>(0.0);
-  if (soft_nms_sigma > static_cast<T>(0.0)) {
-    scale = static_cast<T>(-0.5) / soft_nms_sigma;
```

Comment:
It turns out changing from -0.5 to -1.0 causes failures in some existing applications.

Could you update the PR to use -0.5 and update the tests?  After you update the PR, I can try to patch it and run all internal tests.